data/tekton-pipelines/okd/windows11-installer-example-pipelinerun.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: win11
+  namespace: kubevirt
+  labels:
+    tekton.dev/pipeline: win11
+spec:
+  status: "PipelineRunPending"
+  params:
+    - name: winImageDownloadURL
+      value: >-
+       <INSERT_WIN_11_ISO_URL>
+    - name: autounattendConfigMapName
+      value: windows11-autounattend
+    - name: virtioContainerDiskName
+      value: >-
+        quay.io/kubevirt/virtio-container-disk@sha256:3e88b9bde8792ba21d6cdaad70c79d5e9b339ef676402e932c6fcc2bda100d04
+    - name: sourceTemplateName
+      value: windows11-desktop-large
+    - name: sourceTemplateNamespace
+      value: openshift
+    - name: installerTemplateName
+      value: windows11-desktop-large-installer
+    - name: allowReplaceInstallerTemplate
+      value: 'true'
+    - name: baseDvName
+      value: win11
+    - name: baseDvNamespace
+      value: kubevirt-os-images
+    - name: isoDVName
+      value: win11
+    - name: allowReplaceISODV
+      value: 'false'
+  pipelineRef:
+    name: windows11-installer
+  timeout: 1h0m0s
+  taskRunSpecs:
+    - pipelineTaskName: "modify-windows-iso-file"
```

Comment:
No `taskRunSpecs` for other tasks?

Reply:
This is needed only for task with mounted volume, others does not need it and can run with default values assigned from k8s api